### ISSUE TITLE
a sample json of workshop data for local testing and ejs implementation of index page need to be done

### ISSUE THREAD

**Issue by devansh016** [devansh016/Eventify#6]

**Comment by LPRegen** [comment]
Hi @devansh016 !
I would like to take this issue but I want to check with you some things.

The JSON has to contain the following information:
 - profileImage 
 - title
 - date
 - hour
 - description
 
 Is it correct? I'm missing something?

**Comment by devansh016** [comment]
yes it is correct. You can confirm it from event-model. @LPRegen

**Comment by LPRegen** [comment]
@devansh016 I tried to create a new branch, and push it but I couldn't, plus I saw that you already created the json. 
I will unassign me from this issue.
